modules/apps/transfer/keeper/relay_test.go

```diff
@@ -140,7 +140,7 @@ func (suite *KeeperTestSuite) TestSendTransfer() {
 				suite.Require().NoError(err) // message committed
 			}
 
-			err = suite.chainA.GetSimApp().TransferKeeper.SendTransfer(
+			_, err = suite.chainA.GetSimApp().TransferKeeper.SendTransfer(
```

Comment:
This is a super nit, but maybe we could not discard the return value and assert not zero in the `expPass` cases?
I always lean towards minimising as many discards as possible.

Reply:
good call, updated